shell/profiling/profiler.h

```diff
@@ -0,0 +1,45 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PROFILING_PROFILER_H_
+#define FLUTTER_SHELL_PROFILING_PROFILER_H_
+
+#include <functional>
+#include <memory>
+
+#include "flutter/fml/task_runner.h"
+#include "flutter/fml/trace_event.h"
+
+namespace flutter {
+
+struct CpuUsageInfo {
```

Comment:
Please add docs in Doxygen format for these fields. Specifically, the following clarifications would help me.
* Is `num_threads` the threads used by the process, shell, or more like `nproc`?
* Is total CPU usage a percentage? Is it normalized to logical core count?

Reply:
done!